codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/middleware/UserAgentMiddleware.kt

```diff
@@ -22,10 +22,11 @@ class UserAgentMiddleware(val settings: SwiftSettings) : MiddlewareRenderable {
     override val position = MiddlewarePosition.BEFORE
 
     override fun render(writer: SwiftWriter, op: OperationShape, operationStackName: String) {
+        writer.write("let apiMetadata = ${AWSClientRuntimeTypes.Core.APIMetadata}(serviceId: serviceName, version: \"${settings.moduleVersion}\")")
```

Comment:
Nit: Since this is a type can we use $N here?

Reply:
updated.